build.gradle

```diff
@@ -2,6 +2,7 @@
 plugins {
     id 'com.android.application' version '7.2.0' apply false
     id 'com.android.library' version '7.2.0' apply false
+    id 'org.jetbrains.kotlin.android' version '1.9.22' apply false
```

Comment:
our doris-android use 1.8 and mobile app use 1.7, not sure we should use 1.9 or lower on example app.